tests/integration/high_availability/test_self_healing.py

```diff
@@ -155,22 +140,27 @@ async def test_freeze_db_process(ops_test: OpsTest):
     # verify that connection is possible
     assert is_connection_possible(config), "❌ Mysqld is paused"
 
-    # verify instance is part of the cluster
-    logger.info("Check if instance in cluster")
-    assert await is_unit_in_cluster(
-        ops_test, primary_unit.name, another_unit.name
-    ), "❌ Unit not online in the cluster"
+    # ensure continuous writes still incrementing for all units
+    async with ops_test.fast_forward():
+        await ensure_all_units_continuous_writes_incrementing(ops_test)
 
+    # ensure that we are able to insert data into the primary and have it replicated to all units
+    database_name, table_name = "test-freeze-db-process", "data"
+    await insert_data_into_mysql_and_validate_replication(ops_test, database_name, table_name)
+    await clean_up_database_and_table(ops_test, database_name, table_name)
```

Comment:
would be good to put `clean_up_database_and_table ` in a fixture to gaurentee that the database and table are cleaned up after the test